modules/base/tool.go

```diff
@@ -143,9 +143,23 @@ func FileSize(s int64) string {
 }
 
 // PrettyNumber produces a string form of the given number in base 10 with
-// commas after every three orders of magnitud
-func PrettyNumber(v int64) string {
-	return humanize.Comma(v)
+// commas after every three orders of magnitude
+func PrettyNumber(i interface{}) string {
+	var value int64
+	switch v := i.(type) {
+	case int:
+		value = int64(v)
+	case int8:
+		value = int64(v)
+	case int16:
+		value = int64(v)
+	case int32:
+		value = int64(v)
+	case int64:
+		value = v
+	}
+
+	return humanize.Comma(value)
```

Comment:
Had to change this because the context variables for milestone counts seem to pass `int` which ends up in this function. Maybe it would be better if we would standardize on int64 for such counts, but that seemed like a much greater effort I didn't want to take for this PR.

Reply:
That would make sense if all of them were int64, btw, do you have any idea why int is chosen instead of `uint`, given that uint can hold more numbers and such counts should never be below zero?

Reply:
I didn't investigate Milestone code further, but I guess this is something we should streamline later, e.g. convert counts from signed to unsigned and to 64-bit. Probably would require db migrations too.

Reply:
Culprit seems to be here:

https://github.com/go-gitea/gitea/blob/main/models/issue_milestone.go#L33

Reply:
I never use `unsigned` in business code, `unsigned` can be overflowed easily to UINT_MAX

`int64` is big enough, it's impossible to exhaust it in real cases.

And in many cases, `int` is also big enough, no real people can open 2G~ issues.